source/MaterialXGenShader/TypeDesc.cpp

```diff
@@ -28,6 +30,18 @@ TypeDescNameMap& typeNameMap()
     return map;
 }
 
+
+    //////////////////////////////////////////////////////////////////////////
```

Comment:
For consistency, I would omit these double lines of forward slashes, and the declarations below should be fully left-aligned, as seen in examples like the following:

https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/source/MaterialXCore/Util.cpp#L14